.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules/*
```

Comment:
They are ignored by default...

Reply:
That's true, but we can always add new files/folders to it when needed. That's the reason for including it there - avoiding empyt file.